install_config/storage_examples/storage_classes_dynamic_provisioning.adoc

```diff
@@ -25,14 +30,18 @@ These examples assume some familiarity with Kubernetes, GCE and Persistent Disks
 [[example1]]
 == Scenario 1: Basic Dynamic Provisioning with Two Types of _StorageClasses_
 
-_StorageClasses_ can be used to differentiate and deliniate storage levels and usages, in this case the cluster-admin or storage-admin will set up two distinct classes of storage in GCE.
+_StorageClasses_ can be used to differentiate and delineate storage levels and
+usages. In this case, the `cluster-admin` or `storage-admin` sets up two
+distinct classes of storage in GCE.
 
-- slow:   cheap, efficient and optimized for sequential data operations (slower reading and writing)
-- fast:   optimized for higher rates of random IOPS and sustained throughput (faster reading and writing)
+- `slow`: Cheap, efficient, and optimized for sequential data operations (slower reading and writing)
+- `fast`: Optimized for higher rates of random IOPS and sustained throughput (faster reading and writing)
 
-By creating these _StorageClasses_, the cluster/storage-admin is allowing users to create claims requesting a particular level or service of _StorageClass_.
+By creating these _StorageClasses_, the `cluster-admin` or `storage-admin`
+allows users to create claims requesting a particular level or service of
+_StorageClass_.
```

Comment:
The times StorageClasses are mentioned here use _'s for markup. Should probably have all or none to be consistent with the other parts of this PR.